fix overly-aggressive selector, make filters scrollable

roth-michael opened the pull request:
If desired, you can also add `scrollbar-gutter: static;` to the `.spell-book .spell-filters` styles. I didn't, since you don't use it elsewhere and I'm not sure what my preference is. This should result in the bottom part of the sidebar (number prepared/max prepared, save/reset/stats/loadouts) always being visible, regardless of window size.

Example of the scrollable filters:
<img width="735" height="509" alt="image" src="https://github.com/user-attachments/assets/391a9c57-d82b-488c-b570-c9f321e41da4" />

Sayshal commented:
@roth-michael go ahead and throw in the gutter CSS rules if you could, I was struggling to figure that one out.

If you are able and willing, the other two most recent bug reports are simple missing imports for `log` and `TEMPLATES` if you're able? Mobile GitHub is a pain.

roth-michael commented:
Added the gutters where I thought it made sense, removed an `overflow-y` that wasn't doing anything (and added one to do what that one was supposed to), and fixed those bugs.
Closes #75 
Closes #77
